content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst

```diff
@@ -0,0 +1,132 @@
+field that allows the creator to specify a certain criteria that a product must meet to pass the
+check.
+
+Create a Pass - Fail quality check
+==================================
+
+There are two distinct ways that *Pass - Fail* quality checks can be created. A single check can be
+manually created. Alternatively, a |QCP| can be configured that automatically creates checks at a
+predetermined interval.
+
+This documentation only details the configuration options that are unique to *Pass - Fail* quality
+checks and |QCPs|. For a full overview of all the configuration options available when creating a
+single check or a |QCP|, see the documentation on :ref:`quality checks
+<quality/quality_management/quality-checks>` and :ref:`quality control points
+<quality/quality_management/quality-control-points>`.
+
+Quality check
+-------------
+
+To create a single *Pass - Fail* quality check, navigate to :menuselection:`Quality --> Quality
+Control --> Quality Checks`, and click :guilabel:`New`. Fill out the new quality check form as
+follows:
+
+- In the :guilabel:`Type` drop-down field, select the :guilabel:`Pass - Fail` quality check type.
+- In the :guilabel:`Team` drop-down field, select the quality team responsible for managing the
+  check.
+- In the :guilabel:`Instructions` text field of the :guilabel:`Notes` tab, enter instructions for
+  how to complete the quality check and the criteria that must be met for the check to pass.
+
+.. image:: pass_fail_check/quality-check-form.png
+   :align: center
+   :alt: A quality check form configured for a Pass - Fail quality check.
+
+Quality Control Point (QCP)
+---------------------------
+
+To create a |QCP| that generates *Pass - Fail* quality checks automatically, begin by navigating to
+:menuselection:`Quality --> Quality Control --> Control Points`, and click :guilabel:`New`. Fill out
+the new |QCP| form as follows:
+
+- In the :guilabel:`Type` drop-down field, select the :guilabel:`Pass - Fail` quality check type.
+- In the :guilabel:`Team` drop-down field, select the quality team responsible for managing the
+  checks created by the |QCP|.
+- In the :guilabel:`Instructions` text field, enter instructions for how to complete the quality
+  check and the criteria that must be met for the check to pass.
+
+.. image:: pass_fail_check/qcp-form.png
+   :align: center
+   :alt: A Quality Control Point (QCP) form configured to create a Pass - Fail quality check.
+
+Process a Pass - Fail quality check
+===================================
+
+Once created, there are multiple ways that *Measure* quality checks can be processed. If a quality
+check is assigned to a specific inventory, manufacturing, or work order, the check can be processed
+on the order itself. Alternatively, a check can be processed from the check's page.
+
+From the check's page
+---------------------
+
+To process a *Measure* quality check from the check's page, begin by navigating to
```

Comment:
```suggestion
To process a *Pass - Fail* quality check from the check's page, begin by navigating to
```

Reply:
John, I want to show you a neat lil trick in VS Code for finding and replacing instances of a word.

1. Click CTRL + F to open the word finder at the top right of your screen
![image](https://github.com/odoo/documentation/assets/23611711/056d5de0-c634-4b8f-966e-ed2bdfbe7a92)
2. Enter the word you'd like to find, i.e. "Measure" and it'll show u 
![image](https://github.com/odoo/documentation/assets/23611711/d1cc5af5-c543-4fad-881d-211ef294a73a)
3. You can manually modify it, but to automate it, click the little downwards arrow in the word finder window, and type the word you'd like to replace "Measure" with
![image](https://github.com/odoo/documentation/assets/23611711/305bfb39-5cb0-407f-90be-38a9122589f5)
4. Then, in the replace text field, type "Pass - Fail" right now, and click enter to replace the first instance of Measure, highlighted in bright colors
![image](https://github.com/odoo/documentation/assets/23611711/ca18a2fe-636e-445a-bc10-6c6ce0a4e067)